mon-pix/tests/unit/components/routes/campaigns/assessment/skill-review_test.js

```diff
@@ -241,6 +241,71 @@ describe('Unit | component | Campaigns | Evaluation | Skill Review', function ()
     });
   });
 
+  describe('#acquiredCertifiableBadges', function () {
+    it('should only return certifiable acquired badges', function () {
+      // given
+      const badges = [
+        { id: 33, isAcquired: true, isCertifiable: true },
+        { id: 34, isAcquired: false, isCertifiable: false },
+      ];
+      component.args.model.campaignParticipationResult.campaignParticipationBadges = badges;
+
+      // when
+      const acquiredBadges = component.acquiredCertifiableBadges;
+
+      // then
+      expect(acquiredBadges).to.deep.equal([{ id: 33, isAcquired: true, isCertifiable: true }]);
+    });
+  });
+
+  describe('#notAcquiredButVisibleCertifiableBadges', function () {
+    it('should only return not acquired certifiable badges', function () {
```

Comment:
"not acquired and visible badges", non ?

Reply:
Sinon le badge 37 remonterait, c'est bien ça ?

Reply:
Cette fonction a été modifiée dans un commit ultérieur.